main.js

```diff
@@ -22,14 +27,31 @@ const pg = new Client({
 });
 pg.connect();
 
+app.use(flash());
+app.use(session({
+  secret : 'super secret key',
+  resave : false,
+  saveUninitialized : false
+}));
+app.use(passport.initialize());
+app.use(passport.session());
+
 app.get('/',function(req,res,next){
     let context = {};
+    if (req.user){
+      context.user = req.user
+    }
+    console.log(context); //remove later
     context.title = "Home";
+    console.log(context);
```

Comment:
Remove the redundant log statements please.

Reply:
yes, i was using those for testing.

Reply:
done.